python/lsst/pipe/tasks/multiBand.py

```diff
@@ -111,12 +116,117 @@ def setDefaults(self):
         self.detection.isotropicGrow = True
         self.detection.background.undersampleStyle = 'REDUCE_INTERP_ORDER'
 
+## \addtogroup LSST_task_documentation
+## \{
+## \page DetectCoaddSourcesTask
+## \ref DetectCoaddSourcesTask_ "DetectCoaddSourcesTask"
+## \copybrief DetectCoaddSourcesTask
+## \}
 
 class DetectCoaddSourcesTask(CmdLineTask):
-    """Detect sources on a coadd
-
-    This operation is performed separately in each band.  The detections from
-    each band will be merged before performing the measurement stage.
+    """!
+\anchor DetectCoaddSourcesTask
```

Comment:
Ah, here's a problem with the links: the name of the anchor here doesn't match the reference above (the latter ends with a ``_``). Looks like this problem recurs in several places in this commit.